Modules/StudyProgramme/test/helpers/ilStudyProgrammeGUIMessagesTest.php

```diff
@@ -28,11 +33,10 @@ public function testMessageFactory() : ilPRGMessageCollection
         return $collection;
     }
     
-    /**
-     * @depends testMessageFactory
-     */
-    public function testCollectionDefaults(ilPRGMessageCollection $collection)
```

Comment:
@klees / @nhaagen I removed the dependency of those tests from `testMessageFactory ` since this does not work with random testing